.circleci/config.yml

```diff
@@ -38,7 +38,6 @@ jobs:
           name: Install dependencies
           command: yarn install
 
-      # TODO - why does the test fail without this.
```

Comment:
because few of the packages depend on other packages e.g they depend on react utils that's why we have to build those packages so that they are available in the test environment